adaptive-insurance/src/Layouts/business-info/BusinessInfoFormLayout.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+import React from "react";
+import QuoteCard from "../../components/policy-coverage/QuoteCard";
```

Comment:
Use the relative path for import 
```suggestion
import QuoteCard from "@/components/policy-coverage/QuoteCard";
```